dts/x86/intel_curie.dtsi

```diff
@@ -136,5 +136,33 @@
 
 			status = "disabled";
 		};
+
+		spi0: spi@0xb0001000 {
+			compatible = "snps,designware-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0xb0001000 0x400>;
+			label = "SPI_0";
+			interrupts = <2 IRQ_TYPE_LEVEL_HIGH 2>;
+			interrupt-parent = <&intc>;
+			interrupt-masq = <0xb0800454>;
+			fifo-depth = <7>;
```

Comment:
where is fifo-depth specified?  dts binding?

Reply:
hum, left over from previous rebase. have to remove that